<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="actionMinimal"/>
    </widget>
    <addaction name="menuLayout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMonitoring_New_Process"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionStop_Monitoring"/>
    <addaction name="separator"/>
    <addaction name="actionRun_as_Administrator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Previous"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_Call"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_RAT_Monitor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionMonitoring_New_Process">
   <property name="text">
    <string>Monitor New Process...</string>
   </property>
   <property name="toolTip">
    <string>Run and Monitor a new process</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionRun_as_Administrator">
   <property name="text">
    <string>Run as Administrator</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStop_Monitoring">
   <property name="text">
    <string>Stop Monitoring</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFind_Previous">
   <property name="text">
    <string>Find Previous</string>
   </property>
  </action>
  <action name="actionGo_to_Call">
   <property name="text">
    <string>Go to Call</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionMinimal">
   <property name="text">
    <string>Minimal</string>
   </property>
  </action>
  <action name="actionAbout_RAT_Monitor">
   <property name="text">
    <string>About RAT Monitor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
